### ISSUE TITLE
add support for unity 2017/2018

### ISSUE THREAD

**Issue by wledfor2** [wledfor2/MonoJunkie#1]
add support for unity 2017/2018

simple fix, the DLL name changed, but we'll have to test for both unless we inspect the EXE for the unity version

**Comment by safazi** [comment]
I'm guessing this is the cause of the "Unable to find mono.dll in remote process" error I'm getting.

**Comment by wledfor2** [comment]
Yeah. I checked a few versions of Unity 2017 and 2018 and noticed they keep
changing the filename of the mono DLL. I just added a command line option
to specify the dll name. I forgot to commit it, and will do so when I get
home.

On Jul 15, 2018 2:48 PM, "safazi" <notifications@github.com> wrote:

I'm guessing this is the cause of the "Unable to find mono.dll in remote
process" error I'm getting.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/wledfor2/MonoJunkie/issues/1#issuecomment-405110179>,
or mute the thread
